setup.cfg

```
@@ -0,0 +1,55 @@
+[metadata]
+name = pyhf_benchmark
+version = 0.0.1
+description = Benchmarking of hardware acceleration of pyhf
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/pyhf/pyhf-benchmark
+author = Bo Zheng
+author_email = bozheng96@gmail.com
+license = Apache
+license_file = LICENSE
+keywords = pyhf benchmark
+project_urls =
+    Documentation = https://matthewfeickert.github.io/pyhf_benchmark/
```

```suggestion
    Documentation = https://pyhf.github.io/pyhf-benchmark/
```
I think this got missed in the changes.